operator/src/main/java/oracle/kubernetes/operator/helpers/ResponseStep.java

```diff
@@ -126,6 +147,20 @@ private NextAction doPotentialRetry(Step conflictStep, Packet packet, CallRespon
     return null;
   }
 
+  /**
+   * Resets any retry strategy, such as a failed retry count and invokes the request again. This
+   * will be useful for patterns such as list requests that include a "continue" value.
+   * @param packet Packet
+   * @return Next action for the original request
+   */
+  private NextAction resetResetAndReinvokeRequest(Packet packet) {
```

Comment:
The method name is bit unclear. Should it be something like `resetRetryStrategyAndReinvokeRequest`?

Reply:
Yup... it's a typo. Thanks.